### DISCUSSION TITLE
Chicago Crime Analysis - Level 1 - Jupyter Notebook on Google Colab

### DISCUSSION THREAD

**Discussion by spokplumpen** [DevSkillsHQ/discusssions#18] [Help] [answered]
Hi, I am stuck on point 7 which says "Press the play button on the first query and follow the instructions in the console. That will require granting some permissions to your notebook so that it can query the dataset." 

The thing is when I did press the play button and then visited the URL to authorize, I instead received a page which says "Blocked access: The request from the app is invalid" and "the error 400: redirect_uri_mismatch."

Any idea on how to solve this problem? Thanks!

![error400](https://user-images.githubusercontent.com/47384903/230183737-9e8dc743-75d9-4d30-81e6-13b287726134.jpg)

**Comment by fenske** [comment, marked as answer]
Hi @spokplumpen. I'm Anton from DevSkills.

Sorry about you having to encounter this issue.

It is related to some very recent general changes that have been done to how Google Colab communicates with BigQuery. And unfortunately, it looks like we'll need to come up with a different way for candidates to submit their solutions.

In the meantime, I've updated the instructions for how to submit your solutions. Please find them in the README file of your assignment repository.

Please let me know if you have any other questions.

I wish you the best of luck with the rest of your interview process.
